client/src/components/ViewProfileDetail.js

```diff
@@ -0,0 +1,62 @@
+import React, { useContext } from 'react';
+import styled from 'styled-components';
+import { Card } from 'react-bootstrap';
+import { ProfileContext } from '../context/ProfileContext';
+import avatar from '../assets/icons/avatar.svg';
+import close from '../assets/icons/IconX.svg';
+
+const ViewProfileDetail = () => {
+	const { profile, clearProfile } = useContext(ProfileContext);
```

Comment:
```suggestion
const ViewProfileDetail = ({ profile, clearProfile }) => {
```

Is there any real benefit from using the `ProfileContext` here again, instead of just passing `profile` and `clearProfile` down from the parent, (`<Home/>`) component, where we're also calling `useContext(ProfileContext)`?

I think sometimes passing a prop is the simplest, _purest_ way to create simple components. Context is great if you have a very deeply nested node structure, where you want to avoid passing props all the way through the component tree, especially when the props aren't used by the direct children.